.github/workflows/release.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release to charmhub
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    uses: ./.github/workflows/ci.yaml
+
+  publish:
+    name: Release to CharmHub
+    needs:
+      - build
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check libs
+        uses: canonical/charming-actions/check-libraries@2.1.0
```

Comment:
Please bump all charming-actions to 2.2.2 (there are issues with check-libs).

Reply:
Will implement release to charmhub in a separate PR as Enrico suggested.